code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2611,3 +2611,41 @@
 		var/color
 		CONVERT_PH_TO_COLOR(exposed_atom.reagents.ph, color)
 		exposed_atom.add_atom_colour(color, WASHABLE_COLOUR_PRIORITY)
+
+/datum/reagent/ants
+	name = "Ants"
+	description = "A sample of a lost breed of Space Ants (formicidae bastardium tyrannus), they are well-known for ravaging the living shit out of pretty much anything."
+	reagent_state = SOLID
+	color = "#993333"
+	taste_mult = 1.3
+	taste_description = "tiny legs scuttling down the back of your throat."
+	metabolization_rate = 5 * REAGENTS_METABOLISM //1u per second
+	glass_name = "glass of ants"
+	glass_desc = "Bottoms up...?"
+	ph = 4.6 //Ants contain Formic Acid, meaning they would likely be very acidic themselves if you used them as a reagent
+	var/ant_damage = 0 //How much damage the ants are going to be doing (rises with each tick the ants are in someone's body)
+	var/amount_left = 0 //Tells the debuff how many ants we are being covered with.
```

Comment:
If you're autodocing a variable, the comment goes above the var and uses three forward slashes. 
```suggestion
	/// How much damage the ants are going to be doing (rises with each tick the ants are in someone's body)
	var/ant_damage = 0 
	/// Tells the debuff how many ants we are being covered with.
	var/amount_left = 0 
```

Reply:
Ah, didn't know, I'll keep that in mind and look into fixing the rest of 'em